common/api/presentation-backend.api.md

```diff
@@ -155,9 +155,9 @@ export class PresentationManager {
     activeLocale: string | undefined;
     activeUnitSystem: PresentationUnitSystem | undefined;
     // @deprecated (undocumented)
-    compareHierarchies(requestContext: ClientRequestContext, requestOptions: PresentationDataCompareOptions<IModelDb, NodeKey>): Promise<PartialHierarchyModification[]>;
+    compareHierarchies(requestContext: ClientRequestContext, requestOptions: PresentationDataCompareOptions<IModelDb, NodeKey>): Promise<HierarchyCompareInfo>;

```

Comment:
Doesn't this break a `@public` API?

Reply:
I agree. I think TS allows doing something like the following:
```
myMethod(arg: VersionOne): ResultTypeOne;
myMethod(arg: VersionTwo): ResultTypeTwo;
myMethod(arg: VersionOne | VersionTwo): ResultTypeOne | ResultTypeTwo {
  // return type based on arg type
}
```

Reply:
Missed this as other overload was marked `beta`. Restored API

Reply:
Looks good to me. @pmconne?